src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1232,8 +1232,10 @@ internal enum ErrorCode
         ERR_InvalidAssemblyCulture = 7100,
         ERR_EncReferenceToAddedMember = 7101,
         ERR_MutuallyExclusiveOptions = 7102,
+        WRN_AttributeIgnoredWhenPublicSigning = 7103,
```

Comment:
I think we should move these warning codes down to the bottom. Generally, I prefer to keep ErrorCodes in chronological (or release) order, rather than feature grouping. This prevents new warnings from being associated with old features (which we avoid since adding new warnings to existing code is a breaking change to consumers with /warnaserror).